### PR TITLE
wip: bb blog post

### PR DESCRIPTION
**Pull request by jacobdadams**

### CODE REVIEW

File: _posts/2023-03-03-broadband-fabric-data-challenge.md
Comment (steveoh) [inline _posts/2023-03-03-broadband-fabric-data-challenge.md:19]: **thought**: these fancy dash options are available to you if you so please.
image

File: _posts/2023-03-03-broadband-fabric-data-challenge.md
Comment (steveoh) [inline _posts/2023-03-03-broadband-fabric-data-challenge.md:28]: these can be simplified to .text-left. I don't think flexbox needs to be used.

**Review by stdavis — APPROVED** [review]
Awesome work, as usual.

**Comment by stdavis** [comment]
I'll leave it up to you when you want this merged...

**Comment by jacobdadams** [comment]
Shoot, my update nixed your approved review. Once you approve it will auto-merge.

**Review by stdavis — APPROVED** [review]

**Pull request merged**
